packages/create-resolve-app/src/create_resolve_app.js

```diff
@@ -127,6 +127,15 @@ const runScripts = (appPath, appName, originalDirectory, scriptsPackage) => {
     init.default(appPath, appName, originalDirectory);
 };
 
+const createPackageJson = (appName, appPath) => {
+    const packageJson = {
+        name: appName,
+        version: '0.1.0',
+        private: true
```

Comment:
Should this packages.json include actual reference to lastest resolve-scripts and dependent resolve-libraries version?